charts/shoot-core/components/charts/apiserver-proxy/templates/proxy-configmap.yaml

```diff
@@ -1,13 +1,9 @@
+# TODO(oliver-goetz): Delete this Helm chart in a future version.
```

Comment:
I propose to delete this file right away. This might delete/recreate the ConfigMap, but this should be fine and not affect the running workload. Then you can also delete the helper.

Reply:
I was not entirely sure about the scope of the migration, so I kept everything which is needed that the pods defined by the helm chart are able to restart.Anyway, removing ist also fine with me 😄